### DISCUSSION TITLE
Contribution to ARGO projects

### DISCUSSION THREAD

**Discussion by mksha** [argoproj/argo-rollouts#1890] [General]
Hi Team,

I would love to contribute to ARGO projects, given that i have been using it from a long time so would love to work on some issues and give back to community something.

Please help me how and from where i can start?

**Comment by kostis-codefresh** [comment]
Hello

If you are asking about the technical side of things each project has a contribution page that you can follow https://argoproj.github.io/argo-rollouts/CONTRIBUTING/

If you are asking about what to work on, feel free to sort the issues of the project by "upvotes". Or look at Pull requests, and checkout-confirm them.

You should also join the slack channels and the weekly meetings if you haven't done it already
